DuckDuckGo/MaliciousSiteProtection/Events/MaliciousSiteProtectionEventMapper.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MaliciousSiteProtectionEventMapper.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Common
+import Core
+import MaliciousSiteProtection
+
+enum MaliciousSiteProtectionEventMapper {
+
+    static let debugEvents = EventMapping<MaliciousSiteProtection.Event> { event, _, _, _ in
+        switch event {
+        case .errorPageShown,
+             .visitSite,
+             .iframeLoaded,
+             .settingToggled,
+             .matchesApiTimeout,
+             .failedToDownloadInitialDataSets:
```

Comment:
same as in macOS, thes used to send a Debug Event, now it‘s a regular pixel, just making sure it‘s intended